### PR TITLE
chore(github): add github pull request template

### PR DESCRIPTION
**Pull request by wjehanne**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!-- THE PR TEMPLATE IS NOT AN OPTION. DO NOT DELETE IT, MAKE SURE YOU READ AND EDIT IT! -->
+
+<!--
+    Show us you choose the right branch.
+    Different branches are used for different things :
+    - master is for everything backwards compatible, like patches, features and deprecation notices
+    - 1.x-dev is for deprecation removals and other changes that cannot be done without a BC-break
```
Comment (mremi) [inline .github/PULL_REQUEST_TEMPLATE.md]: does not exist

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    More details here: https://github.com/ekino/sonata/blob/master/CONTRIBUTING.md#the-base-branch
```
Comment (mremi) [inline .github/PULL_REQUEST_TEMPLATE.md]: sonata?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ CHANGELOG
 master
 ------
 
+* Add github pull request template
```
Comment (mremi) [inline CHANGELOG.md]:
```suggestion
* Add GitHub pull request template
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (wjehanne) [reply .github/PULL_REQUEST_TEMPLATE.md]: by the way, this link doesn't work

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ CHANGELOG
 master
 ------
 
+* Add Github pull request template
```
Comment (mremi) [inline CHANGELOG.md]:
```suggestion
* Add GitHub pull request template
```

**Pull request merged**